database/mySql/db.js

```diff
@@ -2,22 +2,65 @@ const mysql = require('mysql');
 const mysqlConfig = require('./config.js');
 
 const db = mysql.createConnection(mysqlConfig);
+db.connect();
 
-module.exports.readId = function (id, cb) {
-  result = {};
-  db.query(`SELECT * FROM images where accommodationId=${id};`, (err, data) => {
-    if (err) {
-      cb(err);
-    } else {
-      result.imgArr = data;
-      db.query(`SELECT name FROM accommodations where id=${id};`, (err, data) => {
-        if (err) {
-          cb(err);
-        } else {
-          result.name = data[0].name;
-          cb(null, result);
-        }
-      });
+// CREATE A NEW RESERVATION FOR A HOUSE
+module.exports.createANewReservationForHouse = function (reservationDetails, cb) {
+  db.query(`INSERT INTO Reservations (adults_amout, childs_amout, infants_amout, check_in_date, check_out_date, house_id) values (${reservationDetails.adults}, ${reservationDetails.children}, ${reservationDetails.infants}, "${reservationDetails.check_in_date}", "${reservationDetails.check_in_date}", ${reservationDetails.id});`,
+    function (error, results, fields) {
+      if (error) {
+        console.log(error);
+        cb(error);
+      } else {
+        cb(results);
+        console.log(`Inserted a new reservation for house id: ${reservationDetails.id}`)
+      }
     }
-  });
-};
\ No newline at end of file
+  );
+};
+
+// GET ALL RESERVATIONS FOR A HOUSE
+module.exports.readAllReservationsFromHouse = function (id, cb) {
+  db.query(
+    `SELECT * from Reservations WHERE house_id = ${id};`,
+    function (error, results, fields) {
+      if (error) {
+        console.log(error);
+        cb(error);
+      } else {
+        cb(results);
+        console.log(`Queried all reservations for house id: ${id}`)
+      }
+    });
+};
+// UPDATE A SPECIFIC RESERVATION
+module.exports.updateReservation = function (reservationDetails, cb) {
+  db.query(
+    `UPDATE Reservations SET check_in_date = "${reservationDetails.check_in_date}", check_out_date = "${reservationDetails.check_out_date}", adults_amout = ${reservationDetails.adults}, childs_amout = ${reservationDetails.children}, infants_amout = ${reservationDetails.infants} WHERE id = ${reservationDetails.reservationId};`,
+
+    function (error, results, fields) {
+      if (error) {
+        console.log(error);
+        cb(error);
+      } else {
+        cb(results);
+        console.log(`Updated reservation ${reservationDetails.reservationId}`)
+      }
+    });
+};
+
+// DELETE A SPECIFIC RESERVATION
+module.exports.deleteReservation = function (reservationDetails, cb) {
```

Comment:
Given you have several methods here, I think the style below looks better, but since it works both ways, your call.
```
module.exports = {
  method1: () => {},
  mothod2: () => {}
}
```